plugins/GSdx/GSRendererOGL.cpp

```diff
@@ -816,6 +816,7 @@ void GSRendererOGL::EmulateTextureSampler(const GSTextureCache::Source* tex)
 	m_ps_sel.tcc = m_context->TEX0.TCC;
 
 	m_ps_sel.ltf = bilinear && shader_emulated_sampler;
+	m_ps_sel.point_sampler = !(bilinear && !shader_emulated_sampler);
```

Comment:
Add a check of AMD GPU. Then I don't like how the condition is written. The global negation is hard to read.
Better write it as
`!bilinear || shader_emulated_sampler`

Just to be sure of the condition, when do you want to enable it ?

Reply:
I want it enabled when Allow 8-bit textures is enabled, I copied the condition from DX